src/main/scala/org/broadinstitute/dsde/vault/VaultServiceActor.scala

```diff
@@ -33,11 +34,13 @@ class VaultServiceActor extends HttpServiceActor with ActorLogging {
 
   // this actor runs all routes
   def receive = runRoute(
-    uBAMIngest.routes ~ uBAMDescribe.routes ~ uBAMRedirect.routes ~
-    analysisIngest.routes ~ analysisDescribe.routes ~ analysisUpdate.routes ~ analysisRedirect.routes ~
-    lookupService.routes ~
-    swaggerService.routes ~ swaggerUiService
-    )
+    swaggerService.routes ~ swaggerUiService ~
+      logOpenAMRequest(log) {
```

Comment:
Because the directive is outside an actor, I switched to using [slf4j directly](https://groups.google.com/forum/#!topic/akka-user/_bIiPKoGJXY). I set the "name" of the logger to "org.broadinstitute.dsde.vault.common.openam.OpenAMRequest", but I'm open to other names too.